apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -143,9 +143,13 @@ export default {
     ...mapState(['user']),
 
     currentPanel() {
-      const panel = this.currentSidebarTab.options?.collaboratorsCurrentPanel
+      const tabOptions = this.currentSidebarTab.options
 
-      return panel || PANEL_SHOW
+      if (tabOptions) {
```

Comment:
Same here, you could go with `if (tabOptions && tabOptions.collaboratorsCurrentPanel) {` to avoid falling back to the same value in two different places.